embedded/sql/options.go

```diff
@@ -75,3 +80,8 @@ func (opts *Options) WithMultiDBHandler(multidbHandler MultiDBHandler) *Options
 	opts.multidbHandler = multidbHandler
 	return opts
 }
+
+func (opts *Options) WithSortBufferSize(size int) *Options {
+	opts.sortBufferSize = size
```

Comment:
we should document it - what it is, what will increase of this value provide, what if we will set to 0

Reply:
Added validity check and description